minio/mc
# Add commands to set, show and remove proxy

## Description

`mc support proxy set|show|remove`

This can be used to configure the proxy to be used to communicate with SUBNET.

## Motivation and Context

Provide a simple high level command to configure proxy for SUBNET communication

## How to test this PR?

`mc support proxy set {alias} {proxy-url}`

should set the given proxy URL in the minio config `subnet proxy`

`mc support proxy show {alias}`

should print the configured proxy on screen.

`mc support proxy remove {alias}`

should remove the minio config `subnet proxy`

## Types of changes

## Checklist:
